src/layouts/AlertLayout.tsx

```diff
@@ -2,14 +2,38 @@ import { LabelProps, RendererProps } from "@jsonforms/core"
 import { Alert } from "antd"
 import { AlertLayoutOptions } from "../ui-schema"
 import { withJsonFormsLabelProps } from "@jsonforms/react"
+import ReactMarkdown from "react-markdown"
 
 export function AlertLayout({ text, uischema }: LabelProps & RendererProps) {
   const options = uischema.options as AlertLayoutOptions
   return (
     <Alert
-      style={{ marginBottom: "24px" }}
-      type={options?.type}
-      message={text}
+      style={{ marginBottom: "24px", borderWidth: 0, padding: "12px" }}
```

Comment:
we really shouldn't hardcode style properties--we can pass them in through uischema.options instead